docs/administering_lagoon/create-project.gql

```diff
@@ -41,16 +22,16 @@ mutation {
   }
 }
 
-# 3. Create a project and assign it the customer and OpenShift
+# 2. Create a project and assign it the OpenShift
 mutation {
   addProject(
     input: {
       # TODO: Fill in the name field
       # This is the project name
       name: ""
-      # TODO: Fill in the customer field
-      # This is the id of the customer to assign to the project
-      customer: 0
+      # TODO: Fill in the private key field
+      # This is the private key for a project, which is used to access the git code
+      privatekey: ""
```

Comment:
let's document here that if no privateKey is added, that Lagoon will create a private key itself and it can be accessed afterwards via loading the project.